package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/SceneImporter.cs

```diff
@@ -518,7 +518,16 @@ public ImportException(string message, System.Exception innerException)
             GameObject go = primMap[pathAndSample.path];
             NativeImporter.ImportObject(scene, go, scene.GetPrimAtPath(pathAndSample.path), importOptions);
             XformImporter.BuildXform(pathAndSample.path, pathAndSample.sample, go, importOptions, scene);
-            CubeImporter.BuildCube(pathAndSample.sample, go, importOptions);
+
+            pxr.UsdSkelSkinningQuery skinningQuery = primMap.SkelCache.GetSkinningQuery(scene.GetPrimAtPath(pathAndSample.path));
+            primMap.SkinningQueries[pathAndSample.path] = skinningQuery;
+            bool skinnedMesh =
+              importOptions.importSkinning
+              && primMap.SkelCache != null
```

Comment:
If SkelCache is null, an error would have been thrown above when calling primMap.SkelCache.GetSkinningQuery.

Reply:
Right, changing that! And actually putting the computation of `skinnedMesh` in a function.